src/ElmFormat/World.hs

```diff
@@ -44,9 +47,16 @@ instance World IO where
     writeFile = Prelude.writeFile
     writeUtf8File path content = ByteString.writeFile path $ Text.encodeUtf8 content
 
-    doesFileExist = Dir.doesFileExist
-    doesDirectoryExist = Dir.doesDirectoryExist
+    doesFileExist path = do
+        exists <- try (Dir.doesFileExist path) :: IO (Either IOException Bool)
```

Comment:
This is to avoid exceptions if detecting the version upward fails (because of permissions issues or something).

Maybe you would have preferred using `try` inside the Free Monad?
If so I would need guidance as I'm not sure how to do this with `Free` monad.